viz.yaml

```diff
@@ -75,13 +78,13 @@ fetch:
     scripts: [scripts/fetch/map_data.R , scripts/fetch/utils.R]
     depends: viewbox_limits
   -
-    id: precip_grid_data
```

Comment:
What I'm looking at right now (on the page as I comment) is an example of why I think these ids should be prefixed with the step -- `fetch` in this case. The github diff hides the `fetch` step header above and I have to think / look for some (albeit small) amount of time to figure out what this thing is.